### DISCUSSION TITLE
Embeddings performance

### DISCUSSION THREAD

**Discussion by CircuitCM** [pyannote/pyannote-audio#1450] [Q&A] [answered]
Hi,
Is anyone aware of benefits to using:
https://huggingface.co/pyannote/embedding
Over
https://huggingface.co/speechbrain/spkrec-ecapa-voxceleb ?
It seems like the prebuilt speaker diarization pipeline uses voxceleb over the pyannote embedding, and voxceleb has a lower EER on the same test set. Also pyannote embedding mentions VAD and PLDA, is there a tool that would let me apply PLDA to an embedding pipeline for better performance?

Thanks

**Comment by hbredin** [comment, marked as answer]
The only benefit of pyannote/embedding over speechbrain would be speed -- but I strongly suggest sticking to speechbrain for this part (much better accuracy).

No PLDA is available in pyannote.

**Reply by zh2336** [reply]
I agree with @CircuitCM that pyannote/embedding performs better than speechbrain in practice, at least on my dataset...

**Reply by CircuitCM** [reply]
Actually I have little idea, barely tested it and still working on my async pipeline. I'm simply taking the experts advise 🙂. But this is interesting to hear, what has been your experience @zh2336? For myself I only need to identify the likelihood a previous segment is not the same speaker as the next segment, so no clustering or large classification task. Maybe there are specific tasks that pyannote embedding is enough for.
